contracts/fair-burn/src/contract.rs

```diff
@@ -0,0 +1,454 @@
+use crate::error::ContractError;
+use crate::msg::{ConfigResponse, ExecuteMsg, InstantiateMsg, QueryMsg, SudoMsg};
+use crate::state::{Config, CONFIG};
+
+use cosmwasm_std::{
+    coin, ensure, to_binary, Addr, BankMsg, Binary, Coin, Decimal, Deps, DepsMut, Env, Event,
+    MessageInfo, StdResult, Uint128,
+};
+use cw2::set_contract_version;
+use cw_utils::maybe_addr;
+use sg_std::{create_fund_fairburn_pool_msg, Response, NATIVE_DENOM};
+use std::collections::BTreeMap;
+
+// version info for migration info
+const CONTRACT_NAME: &str = "crates.io:sg-fair-burn";
```

Comment:
Thinking of using this convention going forward:
- contract names: `stargaze-*`
- package names: `sg-*`
So it will be easy to identify which crate is a contract and which is a package / interface.